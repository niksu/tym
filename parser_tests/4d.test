g() :- a(b1, b2).
a(X, Y) :- b(X), c(Y), d().
a(X, bla) :- b(X), c(X), f(X).
a(X, X).
b(h1).
b(h2).
c(h3).
d().
e(h1).
