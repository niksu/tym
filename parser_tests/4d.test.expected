input contents |g() :- a(b1, b2).
a(X, Y) :- b(X), c(Y), d().
a(X, bla) :- b(X), c(X), f(X).
a(X, X).
b(h1).
b(h2).
c(h3).
d().
e(h1).
|
(no query given)
stringed file contents (size=342, remaining=2658)
|predicate_g() :- predicate_a(constant_b1, constant_b2).
predicate_a(X, Y) :- predicate_b(X), predicate_c(Y), predicate_d().
predicate_a(X, constant_bla) :- predicate_b(X), predicate_c(X), predicate_f(X).
predicate_a(X, X).
predicate_b(constant_h1).
predicate_b(constant_h2).
predicate_c(constant_h3).
predicate_d().
predicate_e(constant_h1).|
stringed file contents strlen=341
